packages/ui/src/upper-bar-manager.tsx

```diff
@@ -126,13 +143,15 @@ export class UpperBarManager {
     //       }
     //     }
     // };
+    const undefinedPluginDefaultOrder: number =
+      Math.max(...Object.values(DefaultPluginOrder)) + 10;
```

Comment:
Why +10?

Reply:
What about this?

Reply:
...A plugin that isn't defined in the list and is registered in the player will be in the order that it's registered after the defined plugins
from design requirements (ticket description section 2)

Reply:
But if you want it last then why not just give it max int?

Reply:
because the max itself is the last plugin position, so anyway i need to add least 1
so i added by multiplication of 10 to meet the tradition (convention)

Reply:
No, I mean instead of irritating the Array just assign it with